pkg/cli/admin/release/extract_tools.go

```diff
@@ -448,6 +448,18 @@ func (o *ExtractOptions) extractCommand(command string) error {
 			InjectReleaseVersion: true,
 			ArchiveFormat:        "openshift-baremetal-install-linux-%s.tar.gz",
 		},
+		{
+			OS:       "linux",
+			Arch:     targetReleaseArch,
+			Command:  "openshift-install-fips",
```

Comment:
There is an option to change the resulted binary name by using `TargetCommandName` as in https://github.com/openshift/oc/blob/425af75f33def2d4dfedbc6ca8a0465be84e343f/pkg/cli/admin/release/extract_tools.go#L279, if you'd want to. Currently, it extracts `openshift-install-fips`.

Reply:
I think that's what we want to minimise confusion, so I will leave it as is.